widgets/google-map/google-map.php

```diff
@@ -505,11 +531,20 @@ function get_template_variables( $instance, $args ) {
 				}
 			}
 
+			// Work out what we need to set gestureHandling to
+			if ( ( ! $settings['draggable'] && ! $settings['scroll_zoom'] ) || ! $settings['draggable'] ) {
```

Comment:
I just checked this very quickly, so could be wrong, but it seems the output of this is ignoring `$settings['scroll_zoom']`.
I.e. it could be simplified to just `! $settings['draggable']`